openrtb_ext/bid.go

```diff
@@ -19,6 +19,7 @@ type ExtBidPrebid struct {
 	DealTierSatisfied bool                `json:"dealtiersatisfied,omitempty"`
 	Meta              *ExtBidPrebidMeta   `json:"meta,omitempty"`
 	Targeting         map[string]string   `json:"targeting,omitempty"`
+	Targetbiddercode  string              `json:"targetbiddercode,omitempty"`
```

Comment:
Can this be renamed with capitalizations to be `TargetBidderCode`

Reply:
done